spec/views/openingtimes/new.html.erb_spec.rb

```diff
@@ -3,7 +3,9 @@
 RSpec.describe "openingtimes/new", type: :view do
   before do
     assign(:openingtime, Openingtime.new(
-                           day: 1
+                           day: 1,
+                           opens: Tod::TimeOfDay.new(8, 0),
+                           closes: Tod::TimeOfDay.new(17, 0)
                          ))
```

Comment:
You can use the openingtime factory here:
```
assign(:openingtime, create :openingtime)
```

Reply:
Usually, the attributes in the view specs are set manually to be able to check if they are rendered. In order to be consistent, I'd leave it as it is